src/views/quote_view.js

```diff
@@ -0,0 +1,51 @@
+import Backbone from 'backbone';
+import Quote from '../models/quote';
+
+const QuoteView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+    this.listenTo(this.model, 'change', this.render);
+    this.listenTo(this.bus, 'order_sale', this.buySellQuote)
+  },
+  render() {
+    const compiledTemplate = this.template(this.model.toJSON());
+
+    this.$el.html(compiledTemplate);
+
+    this.bus.trigger('quote_change_price', this.model)
+
+    return this
+  },
+
+  events: {
+    'click button.btn-buy': 'buyQuote',
+    'click button.btn-sell': 'sellQuote',
+  },
+
+  buyQuote(event) {
+    this.buySellQuote({ buy: true, symbol: this.model.get('symbol') });
+  },
+
+  sellQuote(event) {
+    this.buySellQuote({ buy: false, symbol: this.model.get('symbol') });
+  },
+
+  buySellQuote(info) {
+    if (info.symbol !== this.model.get('symbol')) {
+      return
+    }
+    let salePrice = info.buy ? this.model.buy() : this.model.sell();
+
+    const attributes = {
+      buy: info.buy,
+      symbol: this.model.get('symbol'),
+      price: salePrice
+    }
+    this.bus.trigger('buy_sell_quote', attributes)
```

Comment:
While it's a bit trickier to accomplish using the event bus approach, I think it would be worthwhile to move this trigger logic into the `Quote` model (basically have it happen at the end of `buy()` and `sell()`.

I prefer that approach because it separates the concerns of how to display and manage this quote portion of the page, from how to handle the changes that come from actions/changes on the model layer. In this case, a `Quote` model should always "speak up" about the fact that it was bought/sold, even if that model isn't connected to a particular view instance.